pkg/feature/types.go

```diff
@@ -27,6 +28,12 @@ type OAuth struct {
 	Hmac string
 }
 
+type AuthorinoConfigs struct {
+	CrName,
```

Comment:
How about simply calling it `Name`?

Reply:
`InstanceName`?
Just joking... Let's call it `Name`.

Reply:
Renamed...